src/main/java/com/amaze/filemanager/fragments/FTPServerFragment.java

```diff
@@ -498,11 +509,11 @@ public void onCheckedChanged(CompoundButton buttonView, boolean isChecked) {
      * @return address at which server is running
      */
     private String getFTPAddressString() {
-        String hostAddress = "";
         InetAddress ia = FTPService.getLocalInetAddress(getContext());
-        if (ia != null)
-            hostAddress = ia.getHostAddress();
-        return "ftp://" + hostAddress  + ":" + FTPService.getPort();
+        if(ia == null){
+            throw new NullPointerException("getFTPAddressString(): getLocalInetAddress returned a null value");
```

Comment:
No need to put the name of the method in the Exception ("getFTPAddressString(): ") as it will appear in the stack trace.